### PR TITLE
mm/sl 5pm-2 - Give sections italic font instead of regular

### PR DESCRIPTION
**Pull request by mmunozvaltierra**
As a user I can see sections in italic so that I can easily distinguish sections from lecture at a glance.

NOTE:
After consulting with an LA, due to the way the code is already implemented, writing tests for this issue would require a large portion of the code to be rewritten to make it testable and is beyond the scope of this story.

### CODE REVIEW

File: javascript/src/main/components/BasicCourseSearch/BasicCourseTable.js [hunk 1]
```diff
@@ -15,7 +15,9 @@ const BasicCourseTable = ({ classes, checks, displayQuarter, allowExport }) => {
   const COLOR_UNAVAILABLE = availabilityColors.UNAVAILABLE;
   const COLOR_CLOSEFULL = availabilityColors.CLOSEFULL;
   const COLOR_AVAILABLELECTUREORCLASSWITHSECTIONS = {backgroundColor: '#CEDEFA'};
-  const COLOR_AVAILABLESECTION = {backgroundColor: '#EDF3FE'};
+  const COLOR_AVAILABLESECTION = {backgroundColor: '#EDF3FE', fontStyle: 'italic'};
+  const COLOR_UNAVAILABLESECTION = {backgroundColor: '#FF8080', fontStyle: 'italic' };
+  const COLOR_CLOSEFULLSECTION = {backgroundColor: '#FFD761', fontStyle: 'italic' };
```
Comment (pconrad) [inline javascript/src/main/components/BasicCourseSearch/BasicCourseTable.js:20]: I wonder whether it's a good idea to put code that changes the font into a variable called `COLOR_AVAILABLESECTION`.

(Hint: No, it isn't. :-) )

What would be better style is to 
1. add a meaningful CSS class such as `className="courseSection"` to the elements that are sections.
2. In the file App.css, add code that defines all sections as having `fontStyle: 'italic'`

This should not be merged in this form.

**Review by pconrad — CHANGES_REQUESTED** [review]
This code should not be merged in the current form.

File: javascript/src/main/components/BasicCourseSearch/BasicCourseTable.js (hunk 1 above)
Comment (sydneylim) [reply javascript/src/main/components/BasicCourseSearch/BasicCourseTable.js:20]: We will create a new issue to address these changes.

**Review by 320trankt — APPROVED** [review]
Issue already addressed by professor Conrad, and since we've decided as a team to address this conflict in a new issue, I'll approve these changes for now so we can create a new issue regarding modifying the changes necessary (hopefully taken up by team members proficient in CSS).

**Review by Christine-ong — APPROVED** [review]
LGTM

**Review by sriya112 — COMMENTED** [review]
LGTM

**Review by sriya112 — APPROVED** [review]
LGTM

**Review by pconrad — APPROVED** [review]
I'm not crazy about this approach, but it's good enough for now.... 
and we can refactor it in https://github.com/ucsb-cs156-s21/proj-ucsb-courses-search/issues/298

**Review by pconrad — APPROVED** [review]
LGTM... tested on qa

**Pull request merged**